ngraph/test/models/onnx/quantization/fake_quantize_from_mobilenet.prototxt

```diff
@@ -0,0 +1,70 @@
+ir_version: 6
+producer_name: "pytorch"
+producer_version: "1.5"
+graph {
+  node {
+    output: "666"
```

Comment:
Really ? :)

Reply:
I cut this fragment from a bigger model, without changing any names :)